src/specification.py

```diff
@@ -5,14 +5,27 @@
 
 from license_expression import Licensing
 import validators
+from flask import flash
+from datetime import datetime
 
 valid_relationship = ('DESCRIBES', 'DESCRIBED_BY', 'CONTAINS', 'CONTAINED_BY', 'GENERATES', 'GENERATED_FROM', 'ANCESTOR_OF', 'DESCENDANT_OF', 'VARIANT_OF', 'DISTRIBUTION_ARTIFACT', 'PATCH_FOR', 'PATCH_APPLIED', 'COPY_OF', 'FILE_ADDED', 'FILE_DELETED',
                       'FILE_MODIFIED', 'EXPANDED_FROM_ARCHIVE', 'DYNAMIC_LINK', 'STATIC_LINK', 'DATA_FILE_OF', 'TEST_CASE_OF', 'BUILD_TOOL_OF', 'DOCUMENTATION_OF', 'OPTIONAL_COMPONENT_OF', 'METAFILE_OF', 'PACKAGE_OF', 'AMENDS', 'PREREQUISITE_FOR', 'HAS_PREREQUISITE', 'OTHER')
 
 valid_delivery = ('SOURCE', 'BINARY', 'ARCHIVE', 'APPLICATION',
                   'AUDIO', 'IMAGE', 'TEXT', 'VIDEO', 'DOCUMENTATION', 'SPDX', 'OTHER', 'NOT-DELIVERED')
 
```

Comment:
if feasible, I think it would be better to move the content of valid_relationship and valid_delivery outside the source code, so that the options are not hardcoded (since they might change in the future).

Reply:
Let's accept it for now.
Zeta, feel free to open an issue about it so that it's in the queue for the future